rpc/core/doc.go

```diff
@@ -3,6 +3,7 @@ Package core defines the Ostracon RPC endpoints.
 
 Ostracon ships with its own JSONRPC library -
 https://github.com/Finschia/ostracon/tree/main/rpc/jsonrpc.
+>>>>>>> eed27adde (docs: Update v0.34.x to prepare for v0.37 (#9244))
```

Comment:
This isn't in tendermint v0.34.22, so is a conflict resolution error in this merge operation.
https://github.com/tendermint/tendermint/blob/v0.34.22/rpc/core/doc.go#L6